utils/firestore.js

```diff
@@ -453,6 +453,38 @@ const removeParticipantsDataDestruction = async () => {
     }
 }
 
+const removeUninvitedParticipants = async () => {
+    try {
+
+        const currSnapshot = await db
+            .collection('participants')
+            .where(fieldMapping.participantMap.uninvitedRecruits.toString(), '==', fieldMapping.yes)
```

Comment:
This just my personal preference but I like destructuring the fieldMapping references or another variable if they get too big. The only problem with this is that is adds more lines. Maybe we can assign `fieldMapping.participantMap.uninvitedRecruits.toString()` to another variable. 

`const uninvitedRecruits = fieldMapping.participantMap.uninvitedRecruits.toString()`

or 

`const { uninvitedRecruits } = fieldMapping.participantMap`

`.where(uninvitedRecruits.toString() , '==', fieldMapping.yes )`

Reply:
Solved